applications/grainGrowth/equations.cc

```diff
@@ -66,16 +66,16 @@ customPDE<dim, degree>::explicitEquationRHS(
   // to set the residual terms until all the residuals have been calculated.
 
   std::vector<scalarvalueType> value_terms;
-  value_terms.resize(userInputs.number_of_variables);
+  value_terms.resize(userInputs.var_attributes.size());
   std::vector<scalargradType> gradient_terms;
-  gradient_terms.resize(userInputs.number_of_variables);
+  gradient_terms.resize(userInputs.var_attributes.size());
```

Comment:
Stylistically, var_attributes should have an attribute for the number of variables. Something like var_attributes.n_variables

Reply:
that's cringe

Reply:
Didn't realize it was a map. Resolved